src/steamship/data/file.py

```diff
@@ -184,15 +181,12 @@ def create(
         )
 
     @staticmethod
-    def list(
-        client: Client,
-        corpus_id: str = None,
-    ):
-        req = File.ListRequest(corpusId=corpus_id)
+    def list(client: Client):
+        req = ListRequest()
         res = client.post(
             "file/list",
             payload=req,
-            expect=File.ListResponse,
+            expect=File.ListResponse,  # TODO (enias): Can I rename this?
```

Comment:
(I really like how you add these comments to your future self in PRs; it's a habit I should learn from)